frontend/src/pages/watch/sermons/index.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import Layout from "../../../components/layout";
+import { Breadcrumb } from "gatsby-plugin-breadcrumb";
+import {
+  BookIcon,
+  CalendarIcon,
+  SeriesIcon,
+  UserIcon,
+} from "../../../components/svgs";
+import { StaticImage } from "gatsby-plugin-image";
+
+const SermonPage = ({ pageContext }) => {
+  const {
+    breadcrumb: { crumbs },
+  } = pageContext;
+  return (
+    <Layout>
+      <div className="px-4 w-full flex items-center flex-col">
+        <div className="max-w-container w-full flex flex-col py-[1.375rem] lg:pt-10 lg:pb-[22.8125rem] gap-y-[2.1875rem] lg:gap-y-[3.75rem] items-center">
+          <div className="w-full pl-[2px] lg:pl-0">
+            <Breadcrumb crumbs={crumbs} crumbSeparator=" > " />
+          </div>
+          <div className="flex flex-col gap-y-[0.75rem] lg:gap-y-[3.75rem] items-center max-w-[61.25rem] w-full pb-[3.98625rem] lg:pb-0">
+            <h1 className="text-2xl lg:text-4xl leading-tighter font-semibold lg:font-bold mb-0">
+              Psalm 133
+            </h1>
+            <div className="flex flex-col lg:flex-row lg:gap-x-10 items-center pt-[0.9375rem] lg:pt-0 gap-y-5 lg:gap-y-0">
+              <div className="flex flex-col gap-y-1 lg:gap-y-10 text-base lg:text-xl font-medium w-full lg:w-auto">
+                <div className="flex gap-x-2">
+                  <CalendarIcon />
+                  <div>August 13, 2023</div>
+                </div>
+                <div className="flex flex-col gap-y-1 lg:gap-y-3">
+                  <div className="flex gap-x-2">
+                    <UserIcon />
+                    <div>Pastor Pete Dahlem</div>
+                  </div>
+                  <div className="flex gap-x-2">
+                    <BookIcon />
+                    <div>Psalm 133</div>
+                  </div>
+                  <div className="flex gap-x-2">
+                    <SeriesIcon />
+                    <div>Selah</div>
+                  </div>
+                </div>
+              </div>
+              <div className="w-full max-w-[45rem]">
+                <div className="relative">
+                  <StaticImage
+                    alt="Psalm 133"
+                    src="../../../images/Sermon-Ad-Selah-April.png"
```

Comment:
Can we add some dimensions limitations to this image in case, since its dimensions might be varied coming from the backend?

Reply:
the css will address this change.